README.rst

```diff
@@ -0,0 +1,55 @@
+====================
+dspace-python-client
+====================
+
+A Python client library for working with the DSpace API. This was written against the `DSpace version 6.x API <https://wiki.lyrasis.org/display/DSDOC6x/REST+API>`_ and will not work with later versions of DSpace.
+
+-----
+Usage
+-----
+
+This project has not been released for active use yet. Check back soon!
+
+------------
+Development
+------------
+
+^^^^^^^^^^^^
+Dependencies
+^^^^^^^^^^^^
+~~~~~~
+Python
+~~~~~~
+
+Version 3.9
+
+~~~~~~
+Poetry
+~~~~~~
+This project uses Poetry for dependency management and packaging.
+
+You only need to install Poetry once, so if you've already installed it for another project you can skip this step. Poetry will automatically pick up the current Python version and use it to create virtualenvs accordingly.
+
+To install poetry:
+
+* Mac/unix::
+
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
+* Windows::
```

Comment:
I'd probably remove the windows instructions unless you've actually tested them or maybe change them to a more generic "google and good luck" kinda thing as that syntax looks really weird (I'm sure it's accurate but also... 🤷🏻 )

This looks pretty boilerplate-y so also feel free to keep it if ya want of course :)

Reply:
I haven't tested them but they're copied directly from the Poetry docs...